### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by rubercuellar**

### CODE REVIEW

**Comment by rubercuellar** [comment]
@ReinaldoAguilar

**Comment by ReinaldoAguilar** [comment]
Ruber The structure has to be clean without code

**Comment by rubercuellar** [comment]
@ReinaldoAguilar done. I removed the files

**Comment by ReinaldoAguilar** [comment]
Checkstyle does not recognize and the findbugs

**Comment by rubercuellar** [comment]
@ReinaldoAguilar done, checkstyle and findbugs are working now

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,62 @@
+group 'org.fundacionjala.projects'
```
Comment (carledriss) [inline build.gradle]: projects ?
or pivotal?

File: settings.gradle [hunk 2]
```diff
@@ -0,0 +1,2 @@
+rootProject.name = 'projects'
```
Comment (carledriss) [inline settings.gradle]: projects ? 
or pivotal ?

**Comment by carledriss** [comment]
@AT-03/at-03 
why several directories have README.MD and .gitignore ?
I think is enough to have only one in the project root.

**Pull request merged**